python/samples/demos/sk_mcp_server/mcp_server_with_sampling.py

```diff
@@ -0,0 +1,111 @@
+# /// script # noqa: CPY001
+# dependencies = [
+#   "semantic-kernel[mcp]",
+# ]
+# ///
+# Copyright (c) Microsoft. All rights reserved.
+import logging
+from typing import Annotated, Any
+
+import anyio
+from mcp import types
+from mcp.server.lowlevel import Server
+from mcp.server.stdio import stdio_server
+
+from semantic_kernel import Kernel
+from semantic_kernel.functions import kernel_function
+from semantic_kernel.prompt_template import InputVariable, KernelPromptTemplate, PromptTemplateConfig
+
+logger = logging.getLogger(__name__)
+
+"""
+This sample demonstrates how to expose your Semantic Kernel instance as a MCP server.
```

Comment:
```suggestion
This sample demonstrates how to expose your Semantic Kernel `kernel` instance as a MCP server.
```
or something similar?